examples/colab-notebook-processing_data_using_SQl_and_local_database/colab-notebook-processing_data_using_SQl_and_local_database.ipynb

```diff
@@ -0,0 +1,303 @@
+      "source": [
+        "The relevant Data Job code is fetched in the sqlite-example-job directory. You can see it after running the below cell.\n",
+        "<br>Alternatively, you can use the available code as template and extend it to make a Data Job that fits your use case."
+      ],
+      "metadata": {
+        "id": "Orkqbxmq6b8V"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!git clone https://github.com/vmware/versatile-data-kit\n",
+        "!cp -r /content/versatile-data-kit/examples/sqlite-processing-example/sqlite-example-job/ sqlite-example-job/\n",
+        "!rm -r versatile-data-kit/"
+      ],
+      "metadata": {
+        "id": "GYCT2cWkqpaL"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## 3. Database"
+      ],
+      "metadata": {
+        "id": "9QrNfgmB7Xul"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We will be using the chinook SQLite database. Here we can download it using the following commands."
+      ],
+      "metadata": {
+        "id": "LU5Dl2ZS7e0t"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!curl https://www.sqlitetutorial.net/wp-content/uploads/2018/03/chinook.zip >> chinook.zip\n",
+        "!unzip chinook.zip\n",
+        "!rm -r chinook.zip"
+      ],
+      "metadata": {
+        "id": "DzZpnw3Zp7F8"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "chinook.db' should now be located in the same directory where the original zip file was downloaded."
+      ],
+      "metadata": {
+        "id": "wmbZGGZp754v"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## 4. Configuration"
+      ],
+      "metadata": {
+        "id": "S9YY589K8KTh"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We have previously installed Versatile Data Kit and the plugins required for the example. Now the path to the database we just downloaded must be declared as an environment variable."
+      ],
+      "metadata": {
+        "id": "isMdKlfv8RjJ"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%env VDK_SQLITE_FILE=chinook.db"
+      ],
+      "metadata": {
+        "id": "bVt_UlegyLQE"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## 5. Data Job"
+      ],
+      "metadata": {
+        "id": "wDPpw8xe8wBP"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The structure of our Data Job is as follows:<br><br>\n",
+        "**sqlite-example-job/**<br>\n",
+        "├── 10_drop_table.sql<br>\n",
+        "├── 20_create_table.sql<br>\n",
+        "├── 30_do_processing.sql<br><br>\n",
```

Comment:
The idea is that with VDK instead of having SQL or python steps as separate files, you can embed them into a single notebook as separate cells. 

Take a look at https://github.com/vmware/versatile-data-kit/wiki/Develop-a-Data-Job-through-the-Jupyter-UI . You can see how you can directly 
```
%%vdksql
select query
```
as SQL step 

or 
```
job_input.execute_query("") 
```
as a python step.